tests/rptest/scale_tests/tiered_storage_single_partition_test.py

```diff
@@ -92,6 +92,8 @@ def throughput_test(self, restart_stress):
             f"Configuring to spill metadata after {spillover_segments} segments"
         )
         self.redpanda.set_cluster_config({
+            'cloud_storage_spillover_manifest_size':
```

Comment:
same, are we disabling spillover here or what? confused since the comment above says it does configure spillover

i believe the confusion is on my side, would appreciate explanation

Reply:
answer here https://github.com/redpanda-data/redpanda/pull/16248/commits/45796ffa204490cf92c4d526984fd51b7d37d53f#r1466262067